src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy

```diff
@@ -103,7 +103,7 @@ class PalantirDockerPlugin implements Plugin<Project> {
                         group = 'Docker'
                         description = "Tags Docker image with tag '${tagName}'"
                         workingDir dockerDir
-                        commandLine 'docker', 'tag', '--force=true', ext.name, computeName(ext.name, tagName)
+                        commandLine 'docker', 'tag', ext.name, computeName(ext.name, tagName)
```

Comment:
Right, we need to keep `--force=true` to deal with the existence of containers locally from prior builds. I'd prefer to keep as-is.